.github/workflows/CI.yaml

```diff
@@ -83,6 +83,15 @@ jobs:
       - name: "Build: Split examples by .opt and .meta"
         run: ./scripts/gear.sh build examples-proc
 
+      - name: "Test: Run RPC tests"
```

Comment:
Better to put integration tests at the end

Since with unit-test (that might show the same problem) it will be easier to reason about error reason